sagemaker-inference-recommender/huggingface-inference-recommender/huggingface-inference-recommender.ipynb

```diff
@@ -0,0 +1,989 @@
+{
```

Comment:
<p>Nit - capitalize <code>Analysis</code>  in title?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3075/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Fixed.